### PR TITLE
Fail build on warning in Zanata

### PR DESCRIPTION
**Pull request by aeng**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zanata/zanata-parent/pull/41" target="_blank">ReviewNinja

File: pom.xml [hunk 1]
```diff
@@ -281,7 +281,7 @@
       </plugin>
 
       <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
+          <artifactId>maven-compiler-plugin</artifactId>
```
Comment (seanf) [inline pom.xml]: What?

**Comment by seanf** [comment]
Otherwise, :+1:

**Comment by aeng** [comment]
Revert this merge. There are many non-zanata related warning in Zanata server.

```
[WARNING] Cannot find annotation method 'description()' in type 'org.richfaces.cdk.annotations.Attribute': class file for org.richfaces.cdk.annotations.Attribute not found
[WARNING] Cannot find annotation method 'value()' in type 'org.richfaces.cdk.annotations.Description': class file for org.richfaces.cdk.annotations.Description not found
[WARNING] Cannot find annotation method 'passThrough()' in type 'org.richfaces.cdk.annotations.Attribute'
[WARNING] Cannot find annotation method 'description()' in type 'org.richfaces.cdk.annotations.Attribute'
[WARNING] Cannot find annotation method 'value()' in type 'org.richfaces.cdk.annotations.Description'
[WARNING] Cannot find annotation method 'description()' in type 'org.richfaces.cdk.annotations.Attribute'
[WARNING] Cannot find annotation method 'value()' in type 'org.richfaces.cdk.annotations.Description'
[WARNING] Cannot find annotation method 'events()' in type 'org.richfaces.cdk.annotations.Attribute'
```

**Pull request merged**